cmake/scripts/darwin_embedded/Install.cmake

```diff
@@ -80,6 +80,21 @@ set_target_properties(${APP_NAME_LC} PROPERTIES XCODE_ATTRIBUTE_CODE_SIGN_IDENTI
                                                 XCODE_ATTRIBUTE_DEVELOPMENT_TEAM "${DEVELOPMENT_TEAM}"
                                                 XCODE_ATTRIBUTE_PROVISIONING_PROFILE_SPECIFIER "${PROVISIONING_PROFILE_APP}")
 
+# Create xcode target that allows to build binary-addons.
+if(CORE_PLATFORM_NAME_LC STREQUAL tvos)
+  if(ADDONS_TO_BUILD)
+    set(_addons "ADDONS=${ADDONS_TO_BUILD}")
+  endif()
+  add_custom_target(binary-addons
+    COMMAND $(MAKE) -C ${CMAKE_SOURCE_DIR}/tools/depends/target/binary-addons clean
+    COMMAND $(MAKE) -C ${CMAKE_SOURCE_DIR}/tools/depends/target/binary-addons VERBOSE=1 V=99
+          INSTALL_PREFIX="${CMAKE_BINARY_DIR}/addons" CROSS_COMPILING=yes ${_addons})
+  if(ENABLE_XCODE_ADDONBUILD)
+    add_dependencies(${APP_NAME_LC} binary-addons)
+  endif()
+  unset(_addons)
+endif()
```

Comment:
Why only add this for tvOS?

Reply:
Was put in the cmakebuildsys PR earlier, but was asked to be pulled out for "discussion".
Because of timelines, i felt it easier to attribute to tvos only, and if its desired for ios, i can remove the platform check at a later date.

Reply:
It wasn't related to the previous PR and still isn't to this one. If you want this feature, open a separate PR, which implements it for all platforms.

Reply:
Its a feature to ease building for the apple tv platform, how is that not relevant here? Just because it could be used for ios is not the point. We are working towards updating ios stuff as well, and thats all still to come, why not add a feature for THIS platform now?

Reply:
It's not a feature for tvOS only and also not for tvOS and iOS only, it is a general feature.
The reason I ask for doing it right from the beginning is, that otherwise it might get left behind.

Reply:
Honestly i dont have time to vet it for macos right now. I know it works for ios, but again, i PR'd this early knowing there would be feedback, and i understand where you are coming from regarding "forgotten" catch up features for other platforms, but the point is to get this in for Alpha no? Not everything will be perfect, and theres plenty of todo's. 

I would rather put it through in its known tested state, and when time permits and proper testing is done, open up the platform check to others.